modules/fedimint-mint-tests/tests/tests.rs

```diff
@@ -196,7 +216,7 @@ async fn sends_ecash_out_of_band_cancel() -> anyhow::Result<()> {
     // FIXME: UserCanceledSuccess should mean the money is in our wallet
     for _ in 0..200 {
         sleep_in_test("sats not in wallet yet", Duration::from_millis(100)).await;
-        if client.get_balance().await == sats(1000) {
+        if client.get_balance().await >= sats(1000) - EXPECTED_MAXIMUM_FEE {
```

Comment:
This approximation ain't great, but probably the best we can do for now. We need APIs to get the fees paid for operations/transactions.